projects/components/src/navigation/nav-item/nav-item.component.ts

```diff
@@ -62,13 +62,19 @@ export class NavItemComponent {
       navType: NavigationParamsType.InApp,
       path: item.matchPaths[0],
       relativeTo: this.activatedRoute,
-      replaceCurrentHistory: item.replaceCurrentHistory
+      replaceCurrentHistory: item.replaceCurrentHistory,
+      queryParams: item.queryParams
     };
 
     if (this.config.pageLevelTimeRangeIsEnabled && this.config.timeRangeResolver) {
+      navParams.queryParams = {};
```

Comment:
You're overwriting the query params here - why? If you're trying to default them, change line 66 to `queryParams: item.queryParams ?? {}`

Reply:
This was a draft pr. was experimenting something with Pato :) Wasn't expecting a review.